type-definitions/immutable.js.flow

```diff
@@ -76,8 +76,10 @@ declare class _Collection<K, +V> implements ValueObject {
   has(key: K): boolean;
   includes(value: V): boolean;
   contains(value: V): boolean;
-  first<NSV>(notSetValue?: NSV): V | NSV;
-  last<NSV>(notSetValue?: NSV): V | NSV;
+  first(): V | void;
+  first<NSV>(notSetValue: NSV): V | NSV;
+  last(): V | void;
+  last<NSV>(notSetValue: NSV): V | NSV;
```

Comment:
I've never used Flow before. Is this the correct way to define these types?

Reply:
Never used flow either.
Honestly, I'd like to remove flow types from the main immutable repository as there is no maintenance on it :/